home.html

```diff
@@ -19,7 +19,158 @@ <h1>Good, Morning, Alex.</h1>
     </header>
 
     <main class="main">
-      <h2>Content</h2>
+      <section>Newsfeed</section>
+      <section class="post">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?faces"
+            alt=""
+          />
+          <h2 class="avatar__heading">Jacob Washington</h2>
+          <span class="avatar__timespan">20m ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
+          <q
```

Comment:
Please read https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/